socrates/lib/site/views/format.jade

```diff
@@ -0,0 +1,170 @@
+        each other. By the end of the conference you'll know by name a few
+        dozens of new faces. You'll have made friends, business connections if
+        you want them, maybe you'll find a mentor, or a kindred spirit for that
+        interesting project you want to work on. Anything can happen. And is
+        bound to happen.
+      p
+        | And
+        b  that
+        |  is what SoCraTes is really all about.
+
+      h3 Schedule
+      p.
+        So now that you know what SoCraTes actually is, here are a few more
+        details about the schedule.
+      dl
+
+        dt Thursday
+        dd.
+          The conference starts at 5pm on Thursday, August 25th 2016.
+          This is the time to get to know new faces or meet old friends, have
+          some drinks at the bar, explore the surrounding area etc.
+          T-Shirts are being distributed.
+        dd.
+          Starting around 6pm the hotel will serve a buffet so that even
+          people who arrived a little later will get something to eat.
+        dd Later around 7.30pm we will officially start the conference with a #{''}
+          +extlink('https://en.wikipedia.org/wiki/The_World_Cafe', 'World Café')
+          | , to help kickstart everyone into a great conference experience.
+
+        dt Friday
+        dd Friday is completely reserved for a full-day #{''}
+          +extlink('http://agilecoachcamp.org/tiki-index.php?page=OpenSpace', 'Open Space')
+          | .
+        dd
+          | The marketplace starts at 9am, facilitated by the outstanding facilitator #{''}
+          +extlink('http://blog.connexxo.com/', 'Pierluigi Pugliese')
+          |.
+          | The first group of sessions will start around 10.30am.
+          | The last session timeslot ends at 6pm, which is also
+          | the time we all meet again in the marketplace room for a joint
+          | retrospective of the day.
+        dd
+          | Around 7pm dinner is available for around 2h, so everyone can start
+          | into the evening at their own pace. Of course there's lots of room for
+          | shared evening activities like board games, making music, hacking
+          | or just talking and having fun. Take a look at #{''}
+          a(href='/wiki/' + (currentYear - 1) + '/evening-activities') last year's activities
+          |  to get an impression of the kind of activities that will happen.
+
+        dt Saturday
+        dd Saturday is also completely reserved for a full-day #{''}
+          +extlink('http://agilecoachcamp.org/tiki-index.php?page=OpenSpace', 'Open Space')
+          | .
+        dd.
+          We start the marketplace again at 9am and the first sessions will
+          start around 10.30am too. The last session timeslot ends at 4.30pm.
+          After that we again all meet for the closing of the conference and a
+          retrospective of the two days. The closing of the conference will most
+          likely end around 5.30pm.
```

Comment:
Here, I would also abbreviate like above, but the last sentence should remain intact (so that everybody knows the overall end time).